scss/widgets/header.scss

```diff
@@ -57,3 +62,15 @@
     right: auto;
   }
 }
+
+// Custom header links
```

Comment:
What do you think about naming the class `dc-custom-headers-links`?

This may allow us to avoid using a comment line